### ISSUE TITLE
No introduction to Open Science 

### ISSUE THREAD

**Issue by InesWuerth** [LIKE-ITN/FAIR-data-and-Open-Science-principles#9]
I just started reading the [online version ](https://like-itn-digitalization.readthedocs.io/en/latest/) and I'm missing an introduction to Open Science. You dive straight in to the reasearch lifecycle. Therefore I'd suggest we swop the chapters on Open Science principles and the lifecycle.

**Comment by nikokaoja** [comment]
@InesWuerth , Research Lifecycle, and Open Science are two separate sections. I am not sure what you mean?

**Comment by AndyClifton** [comment]
I think the request is to change the order of the sections, i.e. swop them. The new order would be ...

 - introduction
 - open science principles
 - research lifecycle.
 - fair data principles
 - ...

Probably best to review by phone in the context of planning the [Version 1.0 release](https://github.com/LIKE-ITN/FAIR-data-and-Open-Science-principles/milestone/2).

**Comment by nikokaoja** [comment]
@AndyClifton I agree, lets plan Version 1.0 and split work. I am aware that v0.1.0 was high-pace / 'quick' thing

**Comment by InesWuerth** [comment]
@AndyClifton thanks for clearifying, that was exactly what I meant. Let's discuss in a phone call for v1.0
